src/web-ui/src/partials/ShopperSelect/pages/SelectShopper.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <h1 class="heading">Select Shopper</h1>
+    <p>
+      Select a shopper to see how the personalized product recommendations change based on the shopper’s browsing
+      history and interests.
+    </p>
+
+    <div class="form-container p-4">
+      <div class="mb-4">Use the controls below to select a shopper from the Retail Demo Store user dataset.</div>
+
+      <form @submit.prevent="onSubmit" class="form">
+        <div class="form-group d-flex align-items-center">
+          <label for="age-range" class="label mr-3 mb-0">1</label>
+          <select class="form-control" id="age-range" placeholder="Select age range" v-model="ageRange">
+            <option value="">Select age range</option>
+            <option value="18-24">18-24 years</option>
+            <option value="25-34">25-34 years</option>
+            <option value="35-44">35-44 years</option>
+            <option value="45-54">45-54 years</option>
+            <option value="54-70">54-70 years</option>
+            <option value="70-and-above">Above 70 years</option>
+          </select>
+        </div>
+
+        <div class="form-group d-flex flex-column">
+          <div class="d-flex align-items-center">
+            <label for="primary-interest" class="label mr-3 mb-0">2</label>
+            <select class="form-control" id="primary-interest" v-model="primaryInterest">
+              <option value="">Select primary interest</option>
+              <option value="footwear">Footwear</option>
+              <option value="housewares">Housewares</option>
+              <option value="apparel">Apparel</option>
+              <option value="jewelry">Jewelry</option>
+              <option value="beauty">Beauty</option>
+              <option value="electronics">Electronics</option>
+              <option value="accessories">Accessories</option>
+              <option value="outdoors">Outdoors</option>
```

Comment:
These should be loaded from the `/categories/all` endpoint on the Products service in case categories change and when we add support for deploying with custom datalogs.

Reply:
Agreed, Thats a good point.

Reply:
Agreed that they should be fetched dynamically. They're hard coded right now because of the status of the dataset updates, which is that products with new categories have been introduced but shoppers with those categories as their persona have not.

Assuming we'll hold off on merging the dataset improvements until shopper profiles with the new categories as persona are added, I can roll this back to dynamically fetching categories.

Reply:
Done